proto/steward/v4/cellar_v2.proto

```diff
@@ -495,6 +568,21 @@ message CellarV2_5 {
             SetAlternativeAssetData set_alternative_asset_data = 17;
             // Represents function `setDropAlternativeAssetData(ERC20 _alternativeAsset)`
             DropAlternativeAssetData drop_alternative_asset_data = 18; 
+
```

Comment:
I believe here we are also missing

`SetRebalanceDeviation`
`SetStrategistPlatformCut`
`IncreaseShareSupplyCap`